docs/fornax_test/830_release_testplan.md

```diff
@@ -0,0 +1,584 @@
+kubectl apply -f tests/edgecluster/data/missions/deployment-to-all.yaml
+```
+
+Step 2: run the following command to verify the mission is deployed and the status of the mission is shown and updated regularly.
+```
+kubectl get missions
+```
+
+Step 3: run the following command, with [edge_cluster_kubeconfig] set to the kubeconfig files of cluster B, C and D respectively,
+```
+kubectl get deployment --kubeconfig=[edge_cluster_kubeconfig]
+```
+Verify that the deployment specified in the mission content is created in the edge clusters of B, C, D. 
+
+
+**Test Case 5: An edge cluster continue to work when disconnected from the cloud**
+
+Continuing from the previous test case, do the following:
+
+Step 1: kill the process of edgecore in cluster B. 
+
+Step 2: run the following command, with [edge_cluster_kubeconfig] set to the kubeconfig files of cluster B, C and D respectively,
+```
+kubectl get deployment --kubeconfig=[edge_cluster_kubeconfig]
+```
+Verify that the deployment created via missions in the previous test case is still active in each edge cluster. 
+
+
+**Test Case 6: An edge cluster conintues to monintor its sub-edge-clusters when disconnected from cloud**
+
+Continuing from the previous test case, do the following:
+
+Step 1: Run command 
+```
+kubectl get edgecluster --kubeconfig=[cluster_B_kubeconfig]
+```
+
+Verify the info of edge cluster C is displayed correctly, just like cluster B is not disconnected from the cloud.
+
+Step 2: Run command 
+```
+kubectl get mission --kubeconfig=[cluster_B_kubeconfig]
+```
+
+Verify the info of mission deployment-to-all is displayed correctly, just like cluster B is not disconnected from the cloud.
+
+**Test Case 7: Deploy workload (job) to edge clusters using mission**
+
+Step 1: Run the command in the root operator machine
+```
+kubectl apply -f tests/edgecluster/data/missions/job-to-all.yaml
+```
+
+Step 2: run the following command to verify the mission is deployed
+```
+kubectl get missions
+```
+
+Step 3: run the following command, with [edge_cluster_kubeconfig] set to the kubeconfig files of cluster B, C and D respectively,
+```
+kubectl get jobs --kubeconfig=[edge_cluster_kubeconfig]
+```
+Verify that the job specified in the mission content is created in each edge cluster. 
+
+
+**Test Case 8: Check mission status using command "kubectl get missions"** 
+
+Continuing from the previous test case,
+
+Step 1: run command and watch the output for 1 minute
+```
+watch kubectl get missions
+```
+
+Step 2: verify the the status of the deployed mission are shown correctly and automatically updated. Make sure the name of the edge cluster (B, B/C, B, C and D) and the status of the mission content(deployment/job) are shown in pairs.
+
+**Test Case 9: Update workload to edge clusters using mission**
+
+Continuing from the previous test case,
+
+Step 1: Note down the number of replicas of the deployment specified in the mission content of tests/edgecluster/data/missions/deployment-to-all.yaml
```

Comment:
why note down the number here?

Reply:
Because it will be changed later. We note it down here for the coming comparison.